using-timescaledb/distributed-hypertables.md

```diff
@@ -48,32 +54,203 @@ CREATE TABLE conditions (
 SELECT create_distributed_hypertable('conditions', 'time', 'location');
 ```
 
->:WARNING: If there are no data nodes available, you will get an error
->when executing `create_distributed_hypertable` and the distributed
->hypertable will not be created.
+>:WARNING: If there are no data nodes configured for the database, you
+>will get an error when executing `create_distributed_hypertable` and
+>the distributed hypertable will not be created. Please review the
+>section on [setting up multi-node][getting-started-multi-node]
+>for information on how to add data nodes.
+
+This creates a multi-dimensional distributed hypertable across all
+data nodes, partitioned along `time` and `location`. With the given
+partitioning configuration, each data node will be responsible for a
+specific subset of the data based on the value of `location`, allowing
+concurrent and parallel execution of a query across the data nodes.
+
+You can now insert data into the distributed hypertable and the data
+will automatically be routed to the correct data node.
+
+### Best Practices for Partitioning
+
+Like a regular hypertable, a distributed hypertable needs to be
+partitioned along a "time" dimension (e.g., a `timestamptz`
+column). However, for most distributed workloads, we recommend
+multi-dimensional partitioning with an additional "space" dimension
+that consistently partitions the data over the data nodes, similar to
+traditional [sharding][sharing]. If your data set has a column called
+something similar to `customerID`, `deviceID`, or `location` (as in
+the example above), and it figures frequently in the `GROUP BY` clause
+of queries, then it is likely a good candidate column for space
+partitioning.
+
+With space partitioning, multi-row inserts can often be split across
+the data nodes for increased insert performance, and queries can be
+parallelized by executing a subset of the query on each data
+node. TimescaleDB contains many optimizations to take advantage of the
+partitioning scheme to make best use of multi-node resources.
+
+When a distributed hypertable works with only time partitioning it
+will likely lead to inserts on only one data node at a time. Assuming
+in-order inserts, only one chunk gets written at a time and each chunk
+is placed on a data node in round-robin fashion. Similarly, queries
+cannot be optimally parallelized unless they cover as many chunk time
+intervals as there are data nodes.
+
+## Inserting into a Distributed Hypertable [](insert)
+
+Just like a regular hypertable, it is important to batch data when
+inserting into a distributed hypertable. Each insert statement is
+often its own transaction, and the overhead and cost of this
+transaction is good to amortize over many rows of data. With a
+distributed hypertable, the transaction has additional costs due to
+the coordination that needs to happen across data nodes (e.g.,
+two-phase commit protocol). In such cases, a single INSERT transaction
```

Comment:
Not sure if this is intended to refer to the statement or the concept. IMHO, it's one of the two below:
```suggestion
two-phase commit protocol). In such cases, a single `INSERT` transaction
```

or 

```suggestion
two-phase commit protocol). In such cases, a single insert transaction
```